src/reducers/combined-updaters.js

```diff
@@ -75,7 +75,6 @@ const combinedUpdaters = null;
 
 export const isValidConfig = config => isObject(config) && isObject(config.config) && config.version;
 export const defaultAddDataToMapOptions = {
-  readOnly: false,
```

Comment:
I noticed if the application customizes kepler initial state with readOnly: true, the custom properties is lost because of this setting